src/ArduinoIoTCloudTCP.cpp

```diff
@@ -581,6 +580,30 @@ void ArduinoIoTCloudTCP::sendPropertiesToCloud()
     }
 }
 
+void ArduinoIoTCloudTCP::sendPropertiesToCloud()
+{
+  sendPropertyContainerToCloud(_property_container);
+}
+
+void ArduinoIoTCloudTCP::sendOTAPropertiesToCloud()
+{
+  PropertyContainer to_property_container;
```

Comment:
Those 2 names (`to_property_container`, `from_property_container`) got me confused on the first time. How about naming them like:
* `to_property_container` -> `ota_property_container`
* `from_property_container` -> `full_property_container`